### PR TITLE
Support String queries.

### PR DESCRIPTION
**Pull request by dblock**
This lets you use tooling such as described in https://github.com/artsy/constellation/pull/50 or https://github.com/artsy/constellation/pull/48. It makes Graphlient really thin, but still much saner IMO than using the raw graphql-client. 

This works well with https://github.com/kumarharsh/graphql-for-vscode/pull/50.

### CODE REVIEW

**Comment by ashkan18** [comment]
Nice! Sorry for less contributions lately, should be able to spend more time on this now. This is awesome! 

In general I'll look into VSCode extension and see if we can do anything there, ideally we can improve tooling to follow our DSL as opposed to support tools in our library. I totally agree that this is still nice though 🙌

**Comment by ashkan18** [comment]
restarted Travis build

File: README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 [![Gem Version](https://badge.fury.io/rb/graphlient.svg)](https://badge.fury.io/rb/graphlient)
 [![Build Status](https://travis-ci.org/ashkan18/graphlient.svg?branch=master)](https://travis-ci.org/ashkan18/graphlient)
 
-A much friendlier Ruby client for consuming GraphQL-based APIs, without all the messy strings. Built on top of your usual [graphql-client](https://github.com/github/graphql-client).
+A friendlier Ruby client for consuming GraphQL-based APIs. Built on top of your usual [graphql-client](https://github.com/github/graphql-client), but with better defaults, and using the [faraday](https://github.com/lostisland/faraday) HTTP client.
```
Comment (mzikherman) [inline README.md]: Lol, dropping **much** :(

**Comment by mzikherman** [comment]
Nice!

**Pull request merged**